modules/packages/LinearAlgebra.chpl

```diff
@@ -1308,6 +1308,47 @@ proc svd(A: [?Adom] ?t) throws
   return (u, s, vt);
 }
 
+/* 
+  Compute the approximate solution to ``A * x = b`` using Jacobi method,
+  iteration will stop when ``maxiter`` is reached or error is smaller than
+  ``tol``, whichever comes first. Return the number of iterations performed.
+  
+  .. note::
+    ``X`` is passed as a reference, meaning the initial solution guess can be
+    stored in ``X`` before calling the procedure, and the approximate solution 
+    will be stored in the same array.
+*/
+proc jacobi(A: [?Adom] ?eltType, ref X: [?Xdom] eltType, 
+            b: [Xdom] eltType, tol, maxiter) {
```

Comment:
I'd really like `tol` and `maxiter` to have some kind of a default value. Those values should be documented as well

Reply:
I'll default it to values that were appropriate for my experience in CUDA, but I'm afraid any explanation will only make it confusing to users. CUDA don't explain much of their default configurations too in their AMGX library, just some conventional/intuitive numbers

Reply:
I don't think we need an explanation for why we have chosen specific values. But I think the values should be there in the documentation. So if I want to play around with them (say decrease tolerance) after a default run, I'd have an idea where to begin.

I agree that these values are not exact science, and more of a convention.